migrationTool/ams/AssetMigrationTracker.cs

```diff
@@ -86,9 +86,13 @@ public async Task<AssetMigrationResult> GetMigrationStatusAsync(BlobContainerCli
                 }
 
                 metadataList.TryGetValue(AssetTypeKey, out assetType);
-
+                
                 metadataList.TryGetValue(ManifestNameKey, out manifestName);
-
+                if (!string.IsNullOrEmpty(manifestName))
```

Comment:
it can be 
if (metaataList.TryGetValue(ManifestNameKey, out manifestName))
{
}